### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by dmundra** [rstudio/flexdashboard#333]
Hi,

The flexdashboard has a few accessibility issues (see Wave results for one of the examples https://wave.webaim.org/report#/https://beta.rstudioconnect.com/jjallaire/htmlwidgets-highcharter/htmlwidgets-highcharter.html) in the default HTML that I think should be addressed:

* In the examples include `lang=en` in the markdown so that 'Language missing or invalid' is taken care of. This one the user can fix in their own markdown files.
* Contrast can be fixed using custom CSS.
* The empty button and links for the menu should be handled in JavaScript as it is harder for the user to modify. I see that for the social menu on mobile the word 'Share' appears. That can be modified to have 'Share' be hidden on desktop and visible on mobile.
* The aria-label `role=menu` can be fixed by defining `role=menuitem` for the links in the menu.

I will try to share a pull request but wanted to note down the issues.

Let me know if you have any questions or suggestions.
